File: backend/order-client/src/main/java/org/andante/orders/dto/OrderEntryOutputDTO.java
package org.andante.orders.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.andante.product.dto.ProductVariantOutputDTO;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
public class OrderEntryOutputDTO {

    private long id;
    private int quantity;
    private OrderOutputDTO order;
    private ProductVariantOutputDTO productVariant;
}

File: frontend/src/store/api/result/dto/order/entry/OrderEntryOutputDTO.ts
import { ProductVariantOutputDTO } from "../../product/variant/ProductVariantOutputDTO";
import { OrderOutputDTO } from "../base/OrderOutputDTO";

export interface OrderEntryOutputDTO {
    id: number,
    quantity: number,
    order: OrderOutputDTO,
    productVariant: ProductVariantOutputDTO,
};